src/test/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpCheckTest.java

```diff
@@ -40,15 +40,15 @@ protected String getPackageLocation() {
     @Test
     public void testGetAcceptableTokens() {
         final RegexpCheck regexpCheck = new RegexpCheck();
-        assertArrayEquals(CommonUtil.EMPTY_INT_ARRAY, regexpCheck.getAcceptableTokens(),
-                "RegexpCheck#getAcceptableTokens should return empty array by default");
+        assertWithMessage("RegexpCheck#getAcceptableTokens should return empty array by default")
+                .that(regexpCheck.getAcceptableTokens()).isEqualTo(CommonUtil.EMPTY_INT_ARRAY);
```

Comment:
Is there `isEmpty` for array ?

Reply:
it is compialble:
```
    public void testGetAcceptableTokens() {
        final RegexpCheck regexpCheck = new RegexpCheck();
        assertWithMessage("RegexpCheck#getAcceptableTokens should return empty array by default")
                .that(regexpCheck.getAcceptableTokens())
                .isEmpty();
    }
```

Reply:
Fixed. I guess CI won't guide me on this.